R/teal_slice.R

```diff
@@ -340,6 +337,38 @@ is.teal_slices <- function(x) { # nolint
 }
 
 
+# JSON utils ------------------------------------------------------------------------------------------------------
+
+teal_slice_to_json <- function(x) {
+  if (!is.null(x$selected)) x$selected <- I(x$selected)
+  if (!is.null(x$choices)) x$choices <- I(x$choices)
+
+  x_json <- jsonlite::toJSON(x, pretty = TRUE, auto_unbox = TRUE, digits = 16, null = "null")
+  strsplit(x_json, split = "\n")[[1]]
```

Comment:
Since the function is called `to_json` it should return a JSON string, not a vector of them. I would rather this line be moved into `trim_character` and the latter be renamed `trim_json`

Reply:
make sense